features/invite_your_team.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Invite your team via email
+
+Background:
+    Given we are on Hipchat Login Page
+    When we enter login
+    And we enter password
+    Then we see Welcome title
+
+Scenario: Invite team via email
+    Given we are on Lobby Page
+    When we click Invite your team
+    And we add the emails
+    And we delete one email
```

Comment:
Maybe for checking deleting email from added emails list should be separate scenario?

Reply:
We dont check, we only delete one of added and click send button